SonarQube.Common/ProjectInfo/ProjectInfo.cs

```diff
@@ -32,6 +32,15 @@ public string ProjectName
         }
 
         /// <summary>
+        /// The project language
+        /// </summary>
+        public string ProjectLanguage
```

Comment:
You could enforce the languages here by using an enum. Serializing it should be easy:  

public enum EmployeeStatus
{
   [XmlEnum(Name = "Single")]
   One,